api/HTMLVideoElement.json

```diff
@@ -610,6 +610,58 @@
             "deprecated": true
           }
         }
+      },
+       "msIsStereo3D": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/msIsStereo3D",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": true
+            },
+             "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": false
+            },
+            
```

Comment:
Extraneous empty line.

Reply:
Thanks... and apologies, I'll get the hang of this sooner or later.